modules/cnf-topology-aware-lifecycle-manager-operator-update.adoc

```diff
@@ -53,17 +53,54 @@ spec:
 <1> The index image URL contains the desired Operator images. If the index images are always pushed to the same image name and tag, this change is not needed.
 <2> Set how frequently the Operator Lifecycle Manager (OLM) polls the index image for new Operator versions with the `registryPoll.interval` field. This change is not needed if a new index image tag is always pushed for y-stream and z-stream Operator updates. The `registryPoll.interval` field can be set to a shorter interval to expedite the update, however shorter intervals increase computational load. To counteract this, you can restore `registryPoll.interval` to the default value once the update is complete.
 <3> Last observed state of the catalog connection. The `READY` value ensures that the `CatalogSource` policy is ready, indicating that the index pod is pulled and is running. This way, {cgu-operator} upgrades the Operators based on up-to-date policy compliance states.
++
+[NOTE]
```

Comment:
This is a lot in a single note. Plus normally we try to avoid putting procedures/steps/actions in notes.

I'd recommend trying to pull this out of the note and making it another substep below. 

It seems worthwhile to be its own step. It could be something as simple as "Wait for OLM to update the subscription status." And then follow with the details about the remediation (if the status doesn't update, or what the issue is that persists). Or there might be a better approach.

But if there are situations where they really can't move on to the next step if they are stuck in the state that this note is talking about, then I think a substep would be more appropriate. And just more user-friendly, not having a note that takes up an entire page.

I'll continue with my review as if it's staying in the note, but I'd recommend another set of eyes if you do switch it to a substep

Reply:
Thanks @bergerhoffer 
Yes I was unsure about this one. Apparently this issue occurs in .08% of Operator updates according to testing, so it's considered an edge case. It's something that should be fixed as an enhancement in future but there's no timeline. 

For a bug, I was hesitant to start rewriting the whole section to satisfy the edge case with the new workflow. And I think the ideal scenario is the older workflow. Otherwise, in the new workflow, users will be adding configurations every time they need to update an Operator, which won't be ideal as the PolicyGenTemplate they are adding to will become unwieldy IMO. I tried a "shorter" note (not by much). If you still feel this breaks too many rules, I think I'd need to break out a troubleshooting module perhaps.